V3.0/UFTP-V3.0-metering.xsd

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified">
+
+    <xs:include schemaLocation="UFTP-V3.0-common.xsd"/>
+    <xs:include schemaLocation="UFTP-V3.0-agr-dso.xsd"/>
+
+    <!-- -->
+    <!-- EAN Type -->
+    <!-- -->
+    <xs:simpleType name="EANType">
+        <xs:restriction base="xs:string">
+            <xs:pattern value="[Ee][0-9]{16}"/>
+        </xs:restriction>
+    </xs:simpleType>
+    <!-- -->
+    <!-- -->
+    <!--  Metering Unit Type -->
+    <!-- -->
+    <xs:simpleType name="MeteringUnitType">
```

Comment:
As already answered via e-mail, yes you can make the change here. Then I'll update the rebased branch

Reply:
I removed the "type" suffix.  I agree that combining the units would be a good idea here.

We didn't plan this change into the change document that was voted on, also this would affect backwards compatibility.  We also have other places in the specification where we have power quantities without the units included in the type (to give one example ContractSettlementISPType.RequestedPower).  Therefore I think this change would be better to be done constitently throughout the whole specification.

Reply:
I created an issue for the Removal of MeteringUnitType: https://github.com/shapeshifter/shapeshifter-specification/issues/20